lib/workers/repository/errors-warnings.ts

```diff
@@ -82,3 +79,29 @@ export function getDepWarningsPR(
     '\n\n';
   return warningText;
 }
+
+export function getDepWarningsDashboard(
+  packageFiles: Record<string, PackageFile[]>
+): string {
+  const { warnings, warningFiles } = getDepWarnings(packageFiles);
+  if (!warnings.length) {
+    return '';
+  }
+
+  let depWarnings = '';
+  warnings.forEach((w) => {
+    const dep = w.replace('Failed to look up dependency ', '');
+    depWarnings += '`' + dep + '`, ';
+  });
```

Comment:
```suggestion
  const depWarnings = warnings
    .map((w) => w.replace('Failed to look up dependency ', ''))
    .map((dep) => '`' + dep + '`')
    .join(', ');
```

What do you think about this change? Then doesn't need trailing slash removed later.

Reply:
you are right, i will push the changes
thanks

Reply:
nice 👍